### ISSUE TITLE
Loader.js issue

### ISSUE THREAD

**Issue by ghost** [casperlamboo/plugin-worker#4]
Hi,

I'm running into the issue of the loader.js file not getting the environment suffix, leading to a 404 when requiring loader.js in the index.js file. I see this when developing locally, without building a build file - transpiling at runtime.
I manually tried adjusting this to use either loader-dev.js or loader-production.js and everything starts working great. 

Is there a better to do this?

**Comment by ghost** [comment]
Right, I've found the issue/todo thread in the systemjs repo [#1345](https://github.com/systemjs/systemjs/issues/1345). 
Sorry, wasn't expecting it there. Will try with one of the potential solutions mentioned.

Thanks!

**Comment by ghost** [comment]
Hitting a brick will it seems. No real or temporary solution provided. 
I'll just make a fork of the repo and hardcode to load the loader-production file for now.

**Comment by jfstephe** [comment]
@casperlamboo did you ever get anywhere with this/[#1345](https://github.com/systemjs/systemjs/issues/1345)?

I needed this systemjs config in the packages section to get it working in a browser:

```
    "worker": {
      map: {
        "./loader.js": "./loader-dev.js"
      }
    }
```
